.gitignore

```diff
@@ -1,32 +1,37 @@
-app/bootstrap.php.cache
-app/cache/*
-!app/cache/.gitkeep
-app/cmsdata/*
-!app/cmsdata/.gitkeep
-app/config/parameters.yml
-app/logs/*
-!app/logs/.gitkeep
-behat-parameters.yml
-bin/
-composer.phar
-vendor/
-web/assets/less/chameleon.css
-web/bundles/
-web/chameleon/mediapool/*
-!web/chameleon/mediapool/.gitkeep
+/var/cache/*
+!/var/cache/.gitkeep
+/var/cmsdata/*
+!/var/cmsdata/.gitkeep
+/config/parameters.yaml
+/var/log/*
+!/var/log/.gitkeep
+/behat-parameters.yml
+/bin/
+/composer.phar
+/public/assets/less/chameleon.css
+/public/chameleon/mediapool/*
+!/public/chameleon/mediapool/.gitkeep
 
-web/chameleon/outbox/*
-!web/chameleon/outbox/static/
-web/chameleon/outbox/static/*
+/public/chameleon/outbox/*
+!/public/chameleon/outbox/static/
```

Comment:
And this rename does influence external services: ie where an apache would look for relevant filest?

Reply:
Yes, this will need to be addressed in docs.